lib/build.py

```diff
@@ -92,6 +88,28 @@ def main(cli_):
       cli.tag = re.sub(r"[^a-z0-9_.-]", "", cli.tag.lower())
       ch.INFO("inferred image name: %s" % cli.tag)
 
+   # --force and friends.
+   if (cli.force_cmd and cli.force == "fakeroot"):
+      ch.FATAL("--force-cmd and --force=fakeroot are incompatible")
+   if (not cli.force_cmd):
+      cli.force_cmd = force.FORCE_CMD_DEFAULT
+   else:
+      cli.force = "seccomp"
+      # convert cli.force_cmd to parsed dict
+      force_cmd = dict()
+      for line in cli.force_cmd:
+         (cmd, args) = force.force_cmd_parse(line)
+         force_cmd[cmd] = args
+      cli.force_cmd = force_cmd
+   ch.VERBOSE("force mode: %s" % cli.force)
+   if (cli.force == "seccomp"):
+      for (cmd, args) in cli.force_cmd.items():
+         ch.VERBOSE("force command: %s" % ch.argv_to_string([cmd] + args))
```

Comment:
I know precisely 💩 about Python, so ignore this if I'm Smoking the Good Stuff...but the spacing/indentation, especially the `if` in line 108, looks wonky to me.  Was that intentional?

Reply:
Yes it is intentional, and most Python programmers do think I'm smoking something. (See also the 3-space indents.)

The reasoning is to make the two clauses and the `and` operator appear as three separate things. I like to imagine moving the `and` up half a line.